src/loader/mod.rs

```diff
@@ -193,6 +190,18 @@ pub trait KernelLoader {
 /// Raw ELF (a.k.a. vmlinux) kernel image support.
 pub struct Elf;
 
+#[cfg(feature = "elf")]
+#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
```

Comment:
nit: we can have these cfgs as `#[cfg(all(feature = "elf", any(target_arch = "x86", target_arch = "x86_64"))]`.

Reply:
Much of these are fixed in #24